### PR TITLE
feat(scripts-gulp): replace lerna with nx

### PR DESCRIPTION
**Pull request by Hotell**
## New Behavior

see PR title

## Related Issue(s)

- Partially implements https://github.com/microsoft/fluentui/issues/30265

### CODE REVIEW

File: scripts/gulp/src/tasks/docs.ts [hunk 1]
```diff
@@ -1,31 +1,32 @@
-import { dest, lastRun, parallel, series, src, task, watch, TaskFunction } from 'gulp';
+import { spawnSync } from 'child_process';
```
Comment (Hotell) [inline scripts/gulp/src/tasks/docs.ts:1]: all these import changes => lint autofix

File: scripts/gulp/src/tasks/docs.ts [hunk 2]
```diff
@@ -127,17 +128,16 @@ task('build:docs:webpack', cb => {
 });
 
 task('build:docs:assets:component:info', cb => {
-  const fluentRoot = path.resolve(findGitRoot(), 'packages', 'fluentui');
```
Comment (Hotell) [inline scripts/gulp/src/tasks/docs.ts:130]: it doesn't matter from what cwd is being lerna/nx run thus simplifing

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
LayoutMinimalPerf.default | [212](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/LayoutMinimalPerf.default.html) | [191](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 1.11:1
ButtonMinimalPerf.default | [92](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ButtonMinimalPerf.default.html) | [84](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 1.1:1
SkeletonMinimalPerf.default | [207](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/SkeletonMinimalPerf.default.html) | [192](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.08:1
DropdownMinimalPerf.default | [1483](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/DropdownMinimalPerf.default.html) | [1390](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1.07:1
HeaderMinimalPerf.default | [221](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/HeaderMinimalPerf.default.html) | [206](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1.07:1
HeaderSlotsPerf.default | [483](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/HeaderSlotsPerf.default.html) | [453](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1.07:1
AlertMinimalPerf.default | [157](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/AlertMinimalPerf.default.html) | [148](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/AlertMinimalPerf.default_base.html) | 1.06:1
CardMinimalPerf.default | [317](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/CardMinimalPerf.default.html) | [300](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/CardMinimalPerf.default_base.html) | 1.06:1
FlexMinimalPerf.default | [158](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/FlexMinimalPerf.default.html) | [149](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/FlexMinimalPerf.default_base.html) | 1.06:1
ItemLayoutMinimalPerf.default | [718](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [680](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 1.06:1
LabelMinimalPerf.default | [220](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/LabelMinimalPerf.default.html) | [208](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1.06:1
PortalMinimalPerf.default | [83](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/PortalMinimalPerf.default.html) | [78](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/PortalMinimalPerf.default_base.html) | 1.06:1
ChatMinimalPerf.default | [438](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ChatMinimalPerf.default.html) | [417](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.05:1
ListMinimalPerf.default | [312](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ListMinimalPerf.default.html) | [296](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ListMinimalPerf.default_base.html) | 1.05:1
ListNestedPerf.default | [328](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ListNestedPerf.default.html) | [313](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ListNestedPerf.default_base.html) | 1.05:1
TextAreaMinimalPerf.default | [289](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TextAreaMinimalPerf.default.html) | [274](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 1.05:1
TreeWith60ListItems.default | [83](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TreeWith60ListItems.default.html) | [79](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.05:1
AttachmentSlotsPerf.default | [641](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/AttachmentSlotsPerf.default.html) | [617](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.04:1
DialogMinimalPerf.default | [449](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/DialogMinimalPerf.default.html) | [433](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1.04:1
GridMinimalPerf.default | [189](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/GridMinimalPerf.default.html) | [182](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/GridMinimalPerf.default_base.html) | 1.04:1
ImageMinimalPerf.default | [232](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ImageMinimalPerf.default.html) | [223](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ImageMinimalPerf.default_base.html) | 1.04:1
ListCommonPerf.default | [384](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ListCommonPerf.default.html) | [370](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ListCommonPerf.default_base.html) | 1.04:1
TooltipMinimalPerf.default | [1262](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TooltipMinimalPerf.default.html) | [1213](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 1.04:1
CarouselMinimalPerf.default | [271](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/CarouselMinimalPerf.default.html) | [262](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1.03:1
ChatDuplicateMessagesPerf.default | [152](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [148](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1.03:1
LoaderMinimalPerf.default | [200](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/LoaderMinimalPerf.default.html) | [194](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1.03:1
ReactionMinimalPerf.default | [214](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ReactionMinimalPerf.default.html) | [208](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.03:1
ButtonSlotsPerf.default | [318](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ButtonSlotsPerf.default.html) | [311](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 1.02:1
DividerMinimalPerf.default | [210](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/DividerMinimalPerf.default.html) | [205](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1.02:1
InputMinimalPerf.default | [525](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/InputMinimalPerf.default.html) | [516](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.02:1
RadioGroupMinimalPerf.default | [260](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [256](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 1.02:1
StatusMinimalPerf.default | [391](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/StatusMinimalPerf.default.html) | [385](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.02:1
ToolbarMinimalPerf.default | [542](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ToolbarMinimalPerf.default.html) | [531](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 1.02:1
AttachmentMinimalPerf.default | [83](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/AttachmentMinimalPerf.default.html) | [82](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.01:1
MenuButtonMinimalPerf.default | [967](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [955](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1.01:1
RefMinimalPerf.default | [109](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/RefMinimalPerf.default.html) | [108](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/RefMinimalPerf.default_base.html) | 1.01:1
SegmentMinimalPerf.default | [198](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/SegmentMinimalPerf.default.html) | [196](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 1.01:1
TableMinimalPerf.default | [233](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TableMinimalPerf.default.html) | [231](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.01:1
TextMinimalPerf.default | [198](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TextMinimalPerf.default.html) | [197](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TextMinimalPerf.default_base.html) | 1.01:1
VideoMinimalPerf.default | [435](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/VideoMinimalPerf.default.html) | [432](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1.01:1
DatepickerMinimalPerf.default | [3492](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/DatepickerMinimalPerf.default.html) | [3482](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1:1
FormMinimalPerf.default | [217](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/FormMinimalPerf.default.html) | [217](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/FormMinimalPerf.default_base.html) | 1:1
ListWith60ListItems.default | [365](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ListWith60ListItems.default.html) | [365](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ListWith60ListItems.default_base.html) | 1:1
SplitButtonMinimalPerf.default | [2244](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [2244](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1:1
ButtonOverridesMissPerf.default | [643](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [651](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 0.99:1
PopupMinimalPerf.default | [344](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/PopupMinimalPerf.default.html) | [347](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/PopupMinimalPerf.default_base.html) | 0.99:1
CustomToolbarPrototype.default | [1446](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/CustomToolbarPrototype.default.html) | [1468](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 0.99:1
CheckboxMinimalPerf.default | [1100](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/CheckboxMinimalPerf.default.html) | [1119](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 0.98:1
DropdownManyItemsPerf.default | [392](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/DropdownManyItemsPerf.default.html) | [399](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 0.98:1
MenuMinimalPerf.default | [483](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/MenuMinimalPerf.default.html) | [492](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/MenuMinimalPerf.default_base.html) | 0.98:1
TableManyItemsPerf.default | [1090](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TableManyItemsPerf.default.html) | [1115](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TableManyItemsPerf.default_base.html) | 0.98:1
TreeMinimalPerf.default | [481](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TreeMinimalPerf.default.html) | [493](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/TreeMinimalPerf.default_base.html) | 0.98:1
AvatarMinimalPerf.default | [105](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/AvatarMinimalPerf.default.html) | [108](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 0.97:1
EmbedMinimalPerf.default | [1862](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/EmbedMinimalPerf.default.html) | [1918](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 0.97:1
ProviderMergeThemesPerf.default | [651](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [668](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 0.97:1
ProviderMinimalPerf.default | [198](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ProviderMinimalPerf.default.html) | [205](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 0.97:1
AccordionMinimalPerf.default | [81](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/AccordionMinimalPerf.default.html) | [84](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 0.96:1
RosterPerf.default | [1510](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/RosterPerf.default.html) | [1569](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/RosterPerf.default_base.html) | 0.96:1
AnimationMinimalPerf.default | [289](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/AnimationMinimalPerf.default.html) | [304](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 0.95:1
SliderMinimalPerf.default | [719](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/SliderMinimalPerf.default.html) | [762](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/SliderMinimalPerf.default_base.html) | 0.94:1
IconMinimalPerf.default | [379](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/IconMinimalPerf.default.html) | [402](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/IconMinimalPerf.default_base.html) | 0.94:1
ChatWithPopoverPerf.default | [178](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ChatWithPopoverPerf.default.html) | [192](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 0.93:1
BoxMinimalPerf.default | [191](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/BoxMinimalPerf.default.html) | [208](https://fluentuipr.z22.web.core.windows.net/pull/30266/perf-test-northstar/BoxMinimalPerf.default_base.html) | 0.92:1

</details><div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/702a25813f9be17fcda3c6c4d836c9beb28c14ce'>702a25813f9be17fcda3c6c4d836c9beb28c14ce</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUIV0 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

**Review by ling1726 — APPROVED** [review]

**Pull request merged**